list_dependencies/deps.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+import subprocess
+import json
+
+def format_list(mylist):
+    if len(mylist) < 2:
+        return str(mylist)[1:-1]
+    else:
+        return str(mylist[:-1])[1:-1] + " and '" + str(mylist[-1]) + "'"
+
+def get_node_dependencies():
+    node_str = subprocess.run(
+            ['tilt', 'dump', 'engine'], 
+            stdout=subprocess.PIPE,
+        ).stdout.decode('utf-8')
+    node_data = json.loads(node_str)["ManifestTargets"]
+
+    dependencies = {}
+
+    for resInfo in node_data.values():
+        manifest = resInfo["Manifest"]
+        resName = manifest["Name"]
+        resDeps = manifest["ResourceDependencies"]
```

Comment:
would you like to add ResourceDependencies to UIResource :smile: 

but ya, this is fine for now

Reply:
Yes, I'll add it and start a new PR here once that is done.